packages/flutter/lib/src/foundation/debug.dart

```diff
@@ -76,7 +76,7 @@ Future<T> debugInstrumentAction<T>(String description, Future<T> action()) {
 ///
 ///  * [Timeline.startSync], which typically takes this value as its `arguments`
 ///    argument.
-const Map<String, String> timelineWhitelistArguments = <String, String>{
```

Comment:
in #59408 @Hixie deprecated this and introduced a newly named property. Is this potentially breaking?

Reply:
It is not breaking according to our breaking change policy. I didn't have to modify any tests and g3 is also fine. Therefore, I opted to not do the depreciation.

Reply:
SGTM!